src/spec.md

```diff
@@ -0,0 +1,13 @@
+# The Package.json specification
+
+
+## The specification
+
+
+### Name
+
+The `name` is what your thing is called.
+
+Details:
+- The `name` is an optional property.
+- The `name` must be less than or equal to 214 characters.
```

Comment:
Perhaps the existing npm docs can be copied as a starting point.

https://docs.npmjs.com/cli/v9/configuring-npm/package-json

Reply:
yes, I was using part of it as a reference. I can port the docs easily if we agreed on it at https://github.com/wintercg/proposal-package-json/issues/1#issuecomment-1540929817 in a separate PR 😄